apps/developer-hub/src/components/Pages/PriceFeedsLandingPage/index.tsx

```diff
@@ -0,0 +1,162 @@
+import {
+  ChartLine as BarChartIcon,
+  Clock as ClockIcon,
+  Lightning as LightningIcon,
+  Shuffle as MultiChainIcon,
+  Shield as ShieldIcon,
+  Code as CodeIcon,
+  List as ListIcon,
+  Key as KeyIcon,
+  CurrencyDollar as MoneyWavyIcon,
+} from "@phosphor-icons/react/dist/ssr";
+
+import styles from "./index.module.scss";
+import { ProductCard } from "../../ProductCard";
+
+export function PriceFeedsLandingPage() {
+  return (
+    <div className={styles.container}>
+      <div className={styles.section}>
+        <h1 className={styles.title}>Price Feeds</h1>
+        <p className={styles.lead}>
+          Pyth Price Feeds deliver real-time financial market data sources from
+          120+ first-party providers. These providers include leading exchanges,
+          banks, trading firms, and market makers. Additionally, Pyth Price data
+          can be verified on 100+ blockchains.
+        </p>
+      </div>
+
+      <div className={styles.section}>
+        <h2 className={styles.h2}>Product Options</h2>
+        <p className={styles.body}>
+          Pyth offers two main versions of price feeds, each optimized for
+          different use cases:
+        </p>
+
+        <div className={styles.gridTwo}>
+          <ProductCard
+            badge="Pro"
+            badgeColor="bg-blue-600"
+            icon={<LightningIcon size={24} />}
```

Comment:
Minor nit, but generally I don't like passing sizing to icons from html, I instead prefer setting `font-size` in css -- this is after all part of styling, not content.